File: tests/fixtures/swift/global_conditionals.swift
let debug = true
let verbose = false

if debug {
    if verbose {
        print("verbose debug mode")
    }
}

if debug {
    print("debug on")
}

if verbose {
    print("verbose on")
}

if debug {
    if verbose {
        print("both")
    }
}

func helper() -> Int {
    return 42
}

File: tests/fixtures/r/global_conditionals.r
if (Sys.getenv("DEBUG") == "1") {
  options(warn = 2)
}

if (Sys.getenv("VERBOSE") == "1") {
  cat("verbose mode\n")
}

if (Sys.getenv("STRICT") == "1") {
  options(stringsAsFactors = FALSE)
}

helper <- function(x) {
  x + 1
}
